### DISCUSSION TITLE
Welcome to xray-testng-extensions Discussions!

### DISCUSSION THREAD

**Discussion by bitcoder** [Xray-App/xray-testng-extensions#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rmabb123** [comment]
Will 0.5.0 be uploaded to Maven Central?

**Reply by bitcoder** [reply]
My miss; it's now deployed
